### DISCUSSION TITLE
Victron Compatibility

### DISCUSSION THREAD

**Discussion by o-snoopy-o** [ai-republic/bms-to-inverter#51] [Ideas]
Is it possible to implement Victron protocol?
Please advise. Thanks.

**Comment by ai-republic** [comment]
I believe the Victron uses the same protocol as SMA right?

**Reply by mattthemadscientist** [reply]
oh! I didn't know this! that would be epic as I'm hoping to use some Samsung elpm482-0005 batteries with Victron charge controllers and Selectronic inverters using your software. my Raspberry pi 4B just arrived with the 2 channel can hat. 

I'm just working out how to support both the Victron charge controller and selectronic inverter at the same time via CAN
